tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -71,6 +71,23 @@ def read_model(path_to_xml):
 
     func = read_model(input_model + "_tmp.xml")
 
+    if argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update:
+        return func
+
+    # TODO: use ngraph preprocessing (Mean/Scale/ReverseInputChannels) for legacy frontends
+    reverse_input_channels = False
+    if 'reverse_input_channels' in argv:
+        reverse_input_channels = argv.reverse_input_channels
+        argv.reverse_input_channels = False
+    mean_scale_values = {}
+    if 'mean_scale_values' in argv:
+        mean_scale_values = argv.mean_scale_values
+        argv.mean_scale_values = {}
+    scale = None
+    if 'scale' in argv:
+        scale = argv.scale
+        argv.scale = None
```

Comment:
I think it is added by mistake. Maxim has already activated this in the previous PR. @mvafin, take a look here please

Reply:
Yes, this code was added by mistake during conflicts resolving. Thanks, removed it.